src/SFML/Window/Win32/InputImpl.cpp

```diff
@@ -45,7 +45,7 @@ namespace priv
 ////////////////////////////////////////////////////////////
 bool InputImpl::isKeyPressed(Keyboard::Key key)
 {
-    int vkey = 0;
+    int vkey;
```

Comment:
Same as above, why remove the default case, which will cause undefined behavior?